datalake-dr-connector/src/main/java/com/sequenceiq/cloudbreak/datalakedr/config/DatalakeDrConfig.java

```diff
@@ -0,0 +1,52 @@
+package com.sequenceiq.cloudbreak.datalakedr.config;
+
+import javax.annotation.PostConstruct;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class DatalakeDrConfig {
+
+    private static final int DEFAULT_DATALAKE_DR_PORT = 80;
+
+    @Value("${altus.datalakedr.endpoint:}")
```

Comment:
The value should not have `:` as it will initialize it with an empty string. All these definitions should present in the application.yamls instead.